src/__test__/contact.test.js

```diff
@@ -1,63 +1,27 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-// import Enzyme, { shallow } from 'enzyme';
-// import Adapter from 'enzyme-adapter-react-16';
-import contactPage from '../pages/contactPage/contact';
-import aboutPage from '../pages/aboutPage/aboutPage';
+import ContactPage from '../pages/contactPage/contact';
+import AboutPage from '../pages/aboutPage/aboutPage';
 import Faqs from '../pages/FAQpage/FAQPage';
 import { cleanup } from '@testing-library/react';
-// import Footer from '../components/site/Footer';
-// import Navv from '../components/site/Nav';
 
-// Enzyme.configure({adapter: new Adapter()});
 
 
 afterEach(cleanup);
 
 
 it ('>>>> should render all components', () => {
-	const div = document.createElement('div');
-    ReactDOM.render(<contactPage />, div);
-    ReactDOM.unmountComponentAtNode(div);
+  const div = document.createElement('div');
+  ReactDOM.render(<ContactPage />, div);
+  ReactDOM.unmountComponentAtNode(div);
 
-	const div2 = document.createElement('div');
-    ReactDOM.render(<aboutPage />, div2);
-    ReactDOM.unmountComponentAtNode(div2);
+  const div2 = document.createElement('div');
+  ReactDOM.render(<AboutPage />, div2);
+  ReactDOM.unmountComponentAtNode(div2);
 
-	const div3 = document.createElement('div');
-    ReactDOM.render(<Faqs />, div3);
-    ReactDOM.unmountComponentAtNode(div3);
+  const div3 = document.createElement('div');
+  ReactDOM.render(<Faqs />, div3);
+  ReactDOM.unmountComponentAtNode(div3);
 });
```

Comment:
Can you import **ContextWrapper** from `context/AppProvider.js` and try to render all these components (`<ContactPage />, <AboutPage />, <Faqs />`) inside of the ContextWrapper.

>e.g. 
>const div = document.createElement('div');
>ReactDOM.render(`<ContextWrapper><ContactPage /></ContextWrapper>`);
>ReactDOM.unmountComponentAtNode(div);

try this and see if the tests pass i think it might be the problem

Reply:
let me update it

Reply:
ok cool I see it works now